src/genlib_pmap.erl

```diff
@@ -13,6 +13,8 @@
     proc_limit => pos_integer()
 }.
 
+ -dialyzer({nowarn_function, [spawn_fuse/1]}).
```

Comment:
:crying_cat_face: 
Why do we have to mute this instead of `no_return()` in spec?

Reply:
I tried it and it didn't seem to be enough. However turns out there's a simple trick to stop dialyzer from complaining here, see https://github.com/rbkmoney/genlib/pull/28/commits/273327104e85a595b89d1f8a58605756e188ab38.